cmake/OpenCVFindLibsVideo.cmake

```diff
@@ -229,28 +229,27 @@ if(WITH_FFMPEG)
     if(FFMPEG_INCLUDE_DIR)
       set(HAVE_GENTOO_FFMPEG TRUE)
       set(FFMPEG_LIB_DIR "${FFMPEG_INCLUDE_DIR}/../lib" CACHE PATH "Full path of FFMPEG library directory")
-      if(EXISTS "${FFMPEG_LIB_DIR}/libavcodec.a")
-        set(HAVE_FFMPEG_CODEC 1)
-        set(ALIASOF_libavcodec_VERSION "Unknown")
```

Comment:
You should still set the version variables, so that the configuration summary printed by CMake doesn't look broken.